### PR TITLE
Support of app wide config

### PR DESCRIPTION
**Pull request by kirlat**
Includes updates to work with the app-wide configuration that will be downloaded by the components library (see alpheios-project/alpheios-core#140)

Will remove a branch dependency once `alpheios-core` changes are merged to master.

### CODE REVIEW

File: demo/treebank-translation.html [hunk 1]
```diff
@@ -2,48 +2,35 @@
...
+        new Embedded({clientId: 'embed-lib-demo-treebanks', authEnv: null}).activate();
+      }).catch(e => {
+        console.error(`Import of an embedded library dependencies failed: ${e}`)
+      })
+    }).catch(e => {
+      console.error(`Import of an embedded library failed: ${e}`)
+    })
+  });
+</script>
+<div lang="grc" class="alpheios-enabled">
```
Comment (balmas) [inline demo/treebank-translation.html:24]: can you add the translation pieces back?

File: src/embedded.js [hunk 2]
```diff
@@ -152,13 +152,15 @@ export class Embedded {
...
-      app: { version:`${packageVersion}`, buildNumber: BUILD_NUMBER, name: packageDescription },
+      app: { version:`${packageVersion}`, buildBranch: BUILD_BRANCH, buildNumber: BUILD_NUMBER, buildName: BUILD_NAME, name: packageDescription },
       appType: components.Platform.appTypes.EMBEDDED_LIBRARY,
       clientId: this.clientId,
       // Disable text selection on mobile devices
       disableTextSelection: disableTextSelection,
       textLangCode: textLangCode,
-      overrideHelp: overrideHelp
+      overrideHelp: overrideHelp,
+      configServiceUrl: 'https://4wvouc833c.execute-api.us-east-2.amazonaws.com/prod/config',
```
Comment (balmas) [inline src/embedded.js:162]: and update the default url here too?

File: demo/treebank-translation.html (hunk 1 above)
Comment (kirlat) [reply demo/treebank-translation.html]: Sure, sorry for overlooking that

File: src/embedded.js (hunk 2 above)
Comment (kirlat) [reply src/embedded.js]: Done!

**Pull request merged**